source/boundary_temperature/dynamic_core.cc

```diff
@@ -88,29 +88,34 @@ namespace aspect
         {
           prm.declare_entry ("Outer temperature", "0.",
                              Patterns::Double (),
-                             "Temperature at the outer boundary (lithosphere water/air). Units: $\\si{K}$.");
+                             "Temperature at the outer boundary (lithosphere water/air). Units: \\si{\\kelvin}.");
           prm.declare_entry ("Inner temperature", "6000.",
                              Patterns::Double (),
                              "Temperature at the inner boundary (core mantle boundary) at the "
-                             "beginning. Units: $\\si{K}$.");
+                             "beginning. Units: \\si{\\kelvin}.");
           prm.declare_entry ("dT over dt", "0.",
                              Patterns::Double (),
-                             "Initial CMB temperature changing rate. Units: $K/year$.");
+                             "Initial CMB temperature changing rate. "
+                             "Units: \\si{\\kelvin}/year.");
```

Comment:
Ah, that's a shame that they can't do years...

Reply:
Fun fact: The astronomical standard (Julian year) is exactly 31557600 seconds (or 365.25 days), there are separate ISO standards for normal and leap years, and ASPECT has plumped for the average calendar year (31556952 seconds). So I guess we couldn't claim that we use SI units for years anyway!!

The OCD in me now wants to modify ASPECT for the secular change in length of day :P

Reply:
Oh, a rabbit hole!